docs/source/intro/tutorial.rst

```diff
@@ -570,6 +570,16 @@ Depending on which step the workflow is running, you should get something like t
 We can see that the ``MultiplyAddWorkChain`` is currently waiting for its *child process*, the ``ArithmeticAddCalculation``, to finish.
 Check the process list again for *all* processes (You should know how by now!).
 After about half a minute, all the processes should be in the ``Finished`` state.
+You can also see a nice overview of the steps the work chain with the ``verdi process status`` command:
```

Comment:
```suggestion
You can also see a nice overview of the steps of the work chain with the ``verdi process status`` command:
```
Small correction aside, I am not sure if I would phrase it this way. In the context the word "steps" is often used to refer to the steps of the outline. However, what the command mostly shows is the "call stack", or the hierarchy of process calls. Now this term is probably too technical too use, but we could find a better simpler explanation. Note that the outline steps do enter in the output, but they are in between the square brackets at the end of each line. It shows the current step of each work chain, or the last step, in the case the work chain has finished.

Reply:
How about: "You can also see a nice overview of the processes called by the work chain with the `verdi process status` command:"? And then explain after the code snippet that the square brackets at the end of each work chain line represents the step in the outline.